### PR TITLE
Updated Code

### PR DESCRIPTION
**Pull request by nagasrisai**

### CODE REVIEW

**Comment by nagasrisai** [comment]
Added PDF & PNG Download Option.

**Comment by nagasrisai** [comment]
Capture

File: src/components/Header/Header.module.css [hunk 1]
```diff
@@ -90,32 +71,4 @@
...
-}
-
-@media only screen and (max-width: 460px) {
-    
-    .HeaderDrop{
-        flex-wrap: wrap;
-        gap: 2px;
-    }
-
-    .HeaderOpen{
```
Comment (smaranjitghose) [inline src/components/Header/Header.module.css:102]: What is the reason behind this change?

File: src/pages/Editor/containers/editContext.js [hunk 2]
```diff
@@ -21,7 +22,7 @@ const EditContextProvider = (props) => {
   const [bodyValues, setBodyValues] = useState({
     bodySize: null,
     bodyTop: null,
-    bodyLeft: 0,
+    bodyLeft: 10,
```
Comment (smaranjitghose) [inline src/pages/Editor/containers/editContext.js:25]: Why>?

File: src/pages/Sketch/components/Canvas.js [hunk 3]
```diff
@@ -251,63 +88,15 @@ function Canvas() {
...
-                    onClick={() => setType("pen")}>
-                        <FaPencilAlt/>
-                    </div>
-                    <div className={`${styles.shape_box} ${type === 'line' && styles.active_shape_box}`}
-                    onClick={() => setType("line")}>
-                        <FaSlash/>
-                    </div>
-                    <div className={`${styles.shape_box} ${type === 'square' && styles.active_shape_box}`}
-                    onClick={() => setType("square")}>
-                        <FaRegSquare/>
```
Comment (smaranjitghose) [inline src/pages/Sketch/components/Canvas.js:284]: This is not assigned you. Please do not change this

**Review by smaranjitghose — CHANGES_REQUESTED** [review]

**Comment by smaranjitghose** [comment]
Mutliple files which are changed which has nothing to do with the Download Option.
Deploy Preview is failing.

File: src/pages/Sketch/components/Canvas.js (hunk 3 above)
Comment (nagasrisai) [reply src/pages/Sketch/components/Canvas.js:284]: Hi bro, Please install the dom-to-pdf dependency to work.

Thank you

Comment (smaranjitghose) [reply src/pages/Sketch/components/Canvas.js:284]: How is using dom-to-pdf related to the sketch page at all?
